packages/patternfly-4/react-core/src/components/Alert/Alert.tsx

```diff
@@ -36,11 +38,28 @@ export interface AlertProps extends Omit<React.HTMLProps<HTMLDivElement>, 'actio
   isLiveRegion?: boolean;
 }
 
+export const closeBtnAriaTxt = (ariaLabel: string, variantLabel?: string, title?: string) => {
```

Comment:
I think we should use context to pass title and variant from the Alert to the action components.
The only prop that the AlertActionCloseButton should have related to the label should be an aria-label if they want to override the default we provide. I think it's overcomplicating it a bit to have 3 different props for setting the aria-label

Reply:
Updated, take a look and lemme know what you think!